.dep-versions

```diff
@@ -8,11 +8,11 @@ enzyme=v0.0.149
 
 # For a custom PL version, update the package version here and at
 # 'doc/requirements.txt
-pennylane=0.39.0.dev30
+pennylane=0.39.0
```

Comment:
I think it should not be pinned

Reply:
Chatted offline, it shouldn't make a difference for the time being so we'll leave this as is.